scripts/preinstall.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+/* eslint-disable no-console */
+
+const fs = require('fs')
+const path = require('path')
+const semver = require('semver')
+
+const nodeVersion = process.versions.node
+
+const range = JSON.parse(
+  fs.readFileSync(path.join(__dirname, '..', 'package.json'), 'utf8')
```

Comment:
Using a JS file would make this simpler, especially if we want to use the same kind of logic in all the other references to the minimum Node version in the code.

Reply:
How would that be simpler? Having the `package.json` as the source of truth here seems pretty simple.